documentation/src/main/asciidoc/topics/proc_using_custom_encoders.adoc

```diff
@@ -0,0 +1,55 @@
+[id='using-custom-encoders_{context}']
```

Comment:
This whole section can be omitted since Encoders are deprecated and will be removed soon